### ISSUE TITLE
Design Creator

### ISSUE THREAD

**Issue by JesseWilkinson** [nsimoneaux/tiegenerator#10]
Will you be able to buy a tie that you created your own pattern on.. In other words will you be able to make your own pattern?

**Comment by nsimoneaux** [comment]
Absolutely, you can upload images and also use a paint-style tool. Also, randomly generated patterns.
